### ISSUE TITLE
[DraggableScrollableSheet] precision issue in updateSize

### ISSUE THREAD

**Issue by Doldrums** [flutter/flutter#108266]
TLDR; avoid the accumulation of errors by adding a delta

## Description
Hey folks!! i wonder if accuracy problem in `DraggableScrollableController` can be solved without rounding the value from the outside, which does not look like nice. In general, kinda precision bug will be understandable with the screenshots I have attached:
<img src="https://user-images.githubusercontent.com/38398999/180703704-e9b6bab7-f69b-4f76-bd70-e579a040b5f0.jpg" alt="drawing" width="300"/>
<img src="https://user-images.githubusercontent.com/38398999/180703732-abc73db5-9617-45cb-8a57-32ff2ff69713.jpg" width="300"/>
AAR comment: the problem here is that **u do not avoid the accumulation of errors by adding a delta.**

## Raw solution
<img src="https://user-images.githubusercontent.com/38398999/180705523-845cab02-1f60-4442-bbcf-219d53a19d59.png" alt="drawing" width="300"/>

**Comment by huycozy** [comment]
Hi @Doldrums 
> AAR comment: the problem here is that u do not avoid the accumulation of errors by adding a delta.

Could you please describe the issue more clearly with a specific case along with a [completed and minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) so that we may verify this?
Also, please provide the output of `flutter doctor -v` as well.
Thank you!

**Comment by Doldrums** [comment]
> Hi @Doldrums
> 
> > AAR comment: the problem here is that u do not avoid the accumulation of errors by adding a delta.
> 
> Could you please describe the issue more clearly with a specific case along with a [completed and minimal reproducible code sample](https://stackoverflow.com/help/minimal-reproducible-example) so that we may verify this? Also, please provide the output of `flutter doctor -v` as well. Thank you!

hi! at the moment, I decided to move on with rounding on the outside
